RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -0,0 +1,63 @@
+# This is meant for items that have a date of 2023
+
+boolean auto_haveRockGarden()
+{
+	static item rockGarden = $item[Packet Of Rock Seeds];
+	return auto_is_valid(rockGarden) && auto_get_campground() contains rockGarden;
+}
+
+void rockGardenEnd() //broke these out so they aren't handled at the start of everyday but ASAP after numberology
```

Comment:
I've never actually used these items. What's the benefit of not having them in `pickRocks()`?

Reply:
I moved them from pickRocks because pickRocks was put in daily events, which gets called before the day1 call. That messed up the 151st being the first fight for those of us with numberology. I still did the 151st eventually, but I thought that I'd rather break it out instead of running them at the very start of the day. Honestly, the moleman and stalagmite probably aren't wasting a pull on, but if someone really wants to do that, I figured autoscend should be able to handle it.